alerting/src/main/resources/com/amazon/opendistroforelasticsearch/alerting/alerts/alert_mapping.json

```diff
@@ -125,6 +125,17 @@
           "type": "integer"
         }
       }
+    },
+    "agg_alert_content": {
```

Comment:
I missed this the first time but could you update the schema version in the mapping above to `3` and updated the `AlertIndicesIT` calls to `verifyIndexSchemaVersion` so that it references the update schema version?